jsapi/test/api.test.ts

```diff
@@ -82,6 +82,50 @@ describe('inspectAddress', () => {
         message: "Bech32 error: Invalid character(s) in string"
       });
     });
+    it('Typo in Shelley', () => {
+      const addr = "addr1q9rad75p5r9qk9pnczhyqhkqfv5j2hp7lrmwxpusfqmc8pp7y5xc6cr99fsjcw0hqksgwv3enatmz9ulpp6zn2kuskaqmq72h5";
+      return expect(inspectAddress(addr)).rejects.toEqual({
+        error: { code: "decode" },
+        message: "Bech32 error: Invalid character(s) in string:\naddr1q9rad75p5r9qk9pnczhyqhkqfv5j2hp7lrmwxpusfqmc8pp7y5xc6cr99fsjcw0hqksgwv3enatmz9ulpp6zn2kuskaqmq72h5"
+      });
+    });
+    it('Worng Byron', () => {
+      const addr = "DdzFFzCqrht1JCKJpVrCzJQaQ4KxReo5A3UgXR2aAb1pYpkv82YD75";
+      return expect(inspectAddress(addr)).rejects.toEqual({
+        error: { code: "ErrShelley",
```

Comment:
Maybe the error message could be improved.

Reply:
Yes it could be improved.